app/views/user/show.html.erb

```diff
@@ -0,0 +1,237 @@
+                <div class="hidden-xs">Following</div>
+            </button>
+        </div>
+    </div>
+
+        <div class="well">
+      <div class="tab-content">
+        <div class="tab-pane fade in active" id="tab1">
+          <h3>
+            <%= form_tag(suggestions_search_path(@user.id), :method => "post", id: "search-form") do %>
+              <%= text_field_tag :search, params[:search], placeholder: "Search Posts" %>
+              <%= hidden_field_tag :user_id, value: @user.id %>
+              <%= submit_tag "Search" %>
+            <% end %>
+
+          <% if !params["suggestions_search"].nil? %>
+          <% @result = TunesTakeout.search_results(params["suggestions_search"]) %>
+
+            <% count = 1 %>
+
+            <div class="profile-content">
+              <h3></h3>
+              <table class="table table-striped">
+                <thead>
+                  <tr class = "header">
+                    <th>Rank </th>
+                    <th class = "header">Tunes</th>
+                    <th class = "header">Takeout</th>
+                  </tr>
+                </thead>
+                <tbody class="table-body">
+                  <% track = [] %>
+
+                  <% @result.each do |suggestion| %>
+                    <% music = Music.find(suggestion["music_type"].capitalize, suggestion["music_id"]) %>
+                    <% food = Food.yelp_find_by_business(suggestion["food_id"]) %>
+                    <% album_picture = music.images if music.type == "album" %>
+                    <% album_picture = music.images if music.type == "artist" %>
+                    <% album_picture = music.album.images if music.type == "track" %>
+                    <% @id_value = suggestion["music_id"]%>
+
+                    <tr>
+                      <td class="ranking"><%= count %> </td>
+                      <td><img id="<%=suggestion["music_id"]%>" onclick="changeImage('<%=@id_value%>')" src="http://thumb7.shutterstock.com/display_pic_with_logo/1280533/200698931/stock-vector-most-listened-favorite-icon-200698931.jpg" value = "" width="60%" ></td>
+                      <% count += 1 %>
+                      <% if music.type == "album" %> <td><iframe src="https://embed.spotify.com/?uri=spotify%3AalbumA%<%=music.id%>&theme=white&view=coverart" width="300" height="80" frameborder="0" allowtransparency="true"></iframe></td>
+                        <% elsif music.type == "track" %><td><iframe src="https://embed.spotify.com/?uri=spotify:track:<%=music.id%>" width="300" height="80" frameborder="0" allowtransparency="true"></iframe></td>
+                          <% track << music.id %>
+                        <% else %><td class="artist"><iframe src="https://embed.spotify.com/follow/1/?uri=spotify:artist:<%=music.id%>&size=detail&theme=dark" width="300" height="80" scrolling="no" frameborder="0" style="border:none; overflow:hidden;" allowtransparency="true"></iframe></td>
+                      <% end %>
```

Comment:
This is some great functionality! Try to structure the controller/methods in them to keep as much logic out of the views as possible though!